app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,47 @@
+class PostsController < ApplicationController
+  def index
+    @user = User.find(params[:user_id])
+  end
+
+  def show
+    @user = User.find(params[:user_id])
+    @post = Post.find(params[:id])
+    @like_user = User.first
+  end
+
+  def create
+    @post = Post.new(post_params)
+    @post.author_id = current_user.id
+    @post.likes_counter = 0
+    @post.comments_counter = 0
+    return unless @post.save
+
+    redirect_to user_posts_path
+  end
+
+  def new
+    @post = Post.new
+  end
```

Comment:
- [ ] @Fahim-Khan-P great work on your post controller, however it's ruby convention that the new method comes before the create method, kindly interchange their order.

Reply:
Thank you for your feedback! I agree with your suggestion to follow the Ruby convention and have updated the order of the new and create methods in the PostsController accordingly. Thank you for bringing this to my attention.